privmx-endpoint/src/main/cpp/parser.cpp

```diff
@@ -206,6 +206,21 @@ jobject initEvent(JniContextUtils &ctx, std::string type, std::string channel, i
     jfieldID eventConnectionIdFieldID = ctx->GetFieldID(eventCls, "connectionId",
                                                         "Ljava/lang/Long;");
     jfieldID eventChannelFieldID = ctx->GetFieldID(eventCls, "channel", "Ljava/lang/String;");
+
+    jfieldID eventSubscriptionsFieldID = ctx->GetFieldID(eventCls, "subscriptions",
+                                                         "Ljava/util/List;);");
```

Comment:
incorrect field signature
```suggestion
                                                         "Ljava/util/List;");
```